source/boards/data-and-archives.rst

```diff
@@ -0,0 +1,123 @@
+Import your data
+================
+
+You can import data from other tools to use with Boards. [Contribute code](https://www.focalboard.com/contribute/getting-started/) to expand this.
+
+Import from Asana
+-----------------
+
+This node app converts an Asana JSON archive into a ``.focalboard`` archive.
+
+1. Log into your Asana account.
+2. Select the drop-down menu next to the Asana board's name. Then select **Export/Print > JSON**. This will create an archive file which you'll use in Boards.
+3. Save the file locally, e.g. to ``asana.json``.
+4. Open a terminal window on your local machine and navigate to ``focalboard/webapp``.
+5. Run ``npm install``.
+6. Change directory to ``focalboard/import/asana``.
+7. Run ``npm install``.
+8. Run ``npx ts-node importAsana.ts -i <asana.json> -o archive.focalboard``. This generates the following data:
+
+.. code-block::
+   
+    My-MacbookPro:asana macbook$ npx ts-node importAsana.ts -i ~/Downloads/asana.json -o archive.focalboard
+    Board: 1:1 Meeting Agenda Test
+    Card: [READ ME] Instructions for using this project
+    Card: [EXAMPLE TASK] Feedback on design team presentation
+    Card: [EXAMPLE TASK] Finalize monthly staffing plan
+    Card: [EXAMPLE TASK] Review Q2 launch video outline
+    Card: [EXAMPLE TASK] Mentor a peer
+    
+    Found 5 card(s).
+    Exported to archive.focalboard
+
+9. In Boards, open the board you want to use for the export.
+10. Select **Settings > Import archive** and select ``archive.focalboard``.
+11. Select **Upload**.
+12. Return to your board and confirm that your Asana data is now displaying.
+
+If you don't see your Asana data, an the error should be displayed. You can also check log files for errors.
+
+Import scope
+^^^^^^^^^^^^
+
+Currently, the script imports all cards from a single board, including their section (column) membership, names, and notes.
+
+Import from Notion
+------------------
+
+This node app converts a Notion CSV and markdown export into a ``.focalboard`` archive.
+
+1. From a Notion Board, open the **...** menu at the top right corner of the board.
+2. Select `Export` and pick `Markdown & CSV` as the export format.
+3. Save the generated file locally, and unzip the folder.
+4. Open a terminal window on your local machine and navigate to ``focalboard/webapp``.
+5. Run ``npm install``.
+6. Change directory to `focalboard/import/notion`.
+5. Run ``npm install``.
+6. Run ``npx ts-node importNotion.ts -i <path to the notion-export folder> -o archive.focalboard``. This generates the following data:
```

Comment:
@sbishel, is it possible to provide the same generated data as you did for Asana, for the rest of the tools? If it's the same across the board (barring the tool name) let me know and I can replicate/edit as needed. :)